lib/screens/edit_product_screen.dart

```diff
@@ -184,6 +264,7 @@ class _EditProductScreenState extends State<EditProductScreen> {
                       focusNode: _imageUrlFocusNode,
                       onFieldSubmitted: (_) => _saveForm(),
                       validator: validatorImageUrl,
+                      onSaved: onSavedImageUrl,
```

Comment:
redundant since you already have a controller

Reply:
I tried omitting the onSaved function, the edited product does not reflect.